cmd.sh

```diff
@@ -1,9 +1,10 @@
 #!/bin/sh
 
-GUNICORN_WORKERS="${GUNICORN_WORKERS:-8}"
+# All parameters to the script are appended as arguments to `manage.py serve`
 
-poetry run python manage.py collectstatic --noinput
+set -x
 
-wait-for-it.sh "${DJANGO_DATABASE_HOST}":"${DJANGO_DATABASE_PORT}" -t "${WAITFORIT_TIMEOUT}" -- \
-    poetry run python manage.py migrate --no-input && \
-    poetry run gunicorn --workers=$GUNICORN_WORKERS --bind=0.0.0.0:80 timed.wsgi:application
+poetry run python manage.py manage.py collectstatic --noinput
+
+wait-for-it.sh "${DJANGO_DATABASE_HOST}":"${DJANGO_DATABASE_PORT}" -t "${WAITFORIT_TIMEOUT}" --
+     poetry run python manage.py migrate --no-input &&  poetry run python manage.py serve --static --port 80 --req-queue-len "${HURRICANE_REQ_QUEUE_LEN:-250}" $@
```

Comment:
This could be split into multiple lines for better readability
```suggestion
set -e

wait-for-it.sh "${DJANGO_DATABASE_HOST}":"${DJANGO_DATABASE_PORT}" -t "${WAITFORIT_TIMEOUT}"
poetry run python manage.py migrate --no-input
poetry run python manage.py serve --static --port 80 --req-queue-len "${HURRICANE_REQ_QUEUE_LEN:-250}" "$@"
```

The `set -e` ensures the same as `&&`: abort if one command fails. Also, best practice suggests quoting `"$@"`. See "Special Parameters" in the bash manual for details on the distinction between $*, $@, and "$@" — the first and second are rarely what you want in a safe shell script.